packages/workflow/src/records/fhir.ts

```diff
@@ -1413,8 +1414,11 @@ export function mergeBundles<T extends Bundle, R extends Bundle>(
 export async function findTaskFromIdentifier(
   identifier: string
 ): Promise<Bundle<SavedTask>> {
+  const system =
```

Comment:
There was no reference to which identifier is used here where we actually check if the record already exists or not, meaning the searching with `draft-id` didn't bring up this file.

Reply:
Ah so we needed both the sytem & identifier in the query params for this to work. Then it must have never worked previously in that case :thinking:

Reply:
It did work previously as well! I just wanted to make it explicit which identifier this is searching for so that the next time someone uses code search for `draft-id` to find references to the identifier they find this :)